geode-core/src/upgradeTest/java/org/apache/geode/security/AuthExpirationDUnitTest.java

```diff
@@ -85,9 +86,10 @@ public void clientShouldReAuthenticateWhenCredentialExpiredAndOperationSucceed()
     clientVM.invoke(() -> {
       ClientCache clientCache = ClusterStartupRule.getClientCache();
       UpdatableUserAuthInitialize.setUser("user1");
-      ClientRegionFactory clientRegionFactory =
+      assert clientCache != null;
```

Comment:
Please use actual Assertions in tests instead of Java asserts.

Reply:
The reason for the java assert is not to test something but to get rid of a warning in the gutter. I assume that this is a valid use of java asserts?

Reply:
I'm of the mindset that an assertThat(clientCache).isNotNull(); would be better than what is there. That is pretty much the standard in tests. Whether it is to clear a warning or not.